src/index.ts

```diff
@@ -22,15 +28,24 @@ const getActionInput = () => {
 };
 
 export const main = async (): Promise<void> => {
-  const input = getActionInput();
+  const { FOLLOW_SYMBOLIC_LINKS = 'false', CONFIG_LOCATION_GLOB, CALYPTIA_API_KEY } = getActionInput();
```

Comment:
Nit pick: whitespace at end of line

Reply:
what what? This is processed automatically by [Prettier](https://prettier.io/). If is logic we can chat about it, otherwise is Prettier decision.

Reply:
OK fine if it is automated, bit weird it adds random whitespace to the end of a line though...